content/pt/blog/_posts/2020-12-02-dont-panic-kubernetes-and-docker.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: blog
+title: "Não entre em pânico: Kubernetes e Docker"
+date: 2020-12-02
+slug: dont-panic-kubernetes-and-docker
+---
+
+**Authors:** Jorge Castro, Duffie Cooley, Kat Cosgrove, Justin Garrison, Noah Kantrowitz, Bob Killen, Rey Lejano, Dan “POP” Papandrea, Jeffrey Sica, Davanum “Dims” Srinivas
+
+**Translation:** João Brito (Getup)
```

Comment:
It's OK to translate “Authors” and “Translation” (maybe these are the same words in Portuguese?)

Reply:
No, they're not! Got catch @sftim 

@juniorjbn you can feel free to translate authors and translation :D

Reply:
so, now I can translate the translation? 
It's a suggestion or not? @rikatz

Reply:
Like this perhaps:
```suggestion
**Autores / Autoras**: Jorge Castro, Duffie Cooley, Kat Cosgrove, Justin Garrison, Noah Kantrowitz, Bob Killen, Rey Lejano, Dan “POP” Papandrea, Jeffrey Sica, Davanum “Dims” Srinivas

**Localização:** João Brito (Getup)
```
?

Reply:
I would use `Tradução` instead of `Localização`